PFR/WeakPFR.lean

```diff
@@ -1240,7 +1235,7 @@ max (dimension A') (dimension A'') := by
     convert zpow_neg K 17 using 1
     norm_cast
   calc (dimension B : ℝ)
-    _ ≤ (((max (dimension A') (dimension A'')) : ℕ) : ℝ) := by norm_cast
+    _ ≤ (((max (dimension A') (dimension A'')) : ℕ) : ℝ) := Nat.cast_le.mpr hBdim
```

Comment:
That's less robust than what was before

Reply:
Again, thanks for the explanation. I'll do it later.

Reply:
Done. Thanks!